app/views/update/index.phtml

```diff
@@ -14,7 +14,21 @@
 	</p>
 
 	<?php if (!empty($this->message)) { ?>
-	<p class="alert <?php echo $this->message['status'] === 'bad' ? 'alert-error' : 'alert-warn'; ?>">
+    <?php
```

Comment:
Shouldn't these be tabs?

Reply:
Seems so, looked fine over here. Guess I have to change my IDE settings